test/execute.js

```diff
@@ -529,13 +529,60 @@ describe('execute', () => {
         headers: {
           head: 'justTheHead',
         },
-        body: {
+        body: JSON.stringify({
           json: 'rulez'
-        }
+        })
       })
     })
   })
 
+  it('should use the whole parameters object as the parameter value if parameter is in the body', function () {
```

Comment:
See https://github.com/swagger-api/swagger-js/issues/1006.

> Also, if an api endpoint only has one parameter, what do you think about allowing you to call the operation with the interface operation(value) instead of operation({param_name: value})? If an operation only has one parameter, you could check if the argument passed to the operation is an object with a property matching the parameter name (existing behavior) and if not then just use the passed in argument as the value for the parameter (after validating the type). Are there problems with that approach or does that make it too complicated?

Maybe @fehguy can chime in.